src/okareo_api_client/models/scenario_type.py

```diff
@@ -2,6 +2,8 @@
 
 
 class ScenarioType(str, Enum):
+    ABBREVIATION_SHORTENING = "ABBREVIATION_SHORTENING"
```

Comment:
We talked about a different type name. Good to get it in before it starts to populate all the tables. Suggestion: COMMON_CONTRACTIONS